apps/dashboard/src/utils/parseStepVariables.ts

```diff
@@ -1,8 +1,12 @@
+import { Completion } from '@codemirror/autocomplete';
 import type { JSONSchemaDefinition } from '@novu/shared';
+import { DIGEST_VARIABLES } from '../components/variable/utils/digest-variables';
 
 export interface LiquidVariable {
-  type: 'variable';
+  type: 'variable' | 'digest';
```

Comment:
added a digest type custom variable so that we can target it using CSS and add custom icons in the codemirror editor